### DISCUSSION TITLE
Error : Scheduling request failed: releasing RRC connection

### DISCUSSION THREAD

**Discussion by rajugbv** [srsran/srsRAN_Project#381] [General Help]
Hello everyone,

First and foremost, I would like to express my appreciation for the incredible support from the srsran community. Thanks to your assistance, I successfully connected to an older version of srsran, open5gs, and Ubuntu, even managing to transmit a video using VLC. However, I am currently facing difficulties connecting the gNB and UE. I am uncertain about where I may have made a mistake. Could someone please offer guidance and assistance?

![Screenshot from 2023-12-20 15-53-19](https://github.com/srsran/srsRAN_Project/assets/41448193/841fc3ed-a693-4ec1-8783-493beb07b833)

https://drive.google.com/drive/folders/1M9SgLKRHQk22D7fZt_k97nSr96gBPzNq?usp=drive_link

**Comment by pgawlowicz** [comment]
Please set `coreset0_index: 6` in the gnb config.
